packages/openneuro-server/src/graphql/resolvers/dataset.js

```
@@ -181,6 +181,35 @@ export const deleteFile = async (
   }
 }
 
+export const deleteBulk = async (
+  obj,
+  { datasetId, files },
+  { user, userInfo },
+) => {
+  try {
+    await checkDatasetWrite(datasetId, user, userInfo)
+    const deletedFiles = await datalad
+      .deleteBulk(datasetId, files, userInfo)
+      .then(filenames =>
+        Promise.all(filenames.map(filename => new UpdatedFile(filename))),
+      )
```

Looks like this can fail silently if you delete files that don't exist?